Source/Eto.Gtk/Forms/GtkWindow.cs

```diff
@@ -249,7 +249,10 @@ public override void AttachEvent(string id)
 					Control.DeleteEvent += Connector.HandleDeleteEvent;
 					break;
 				case Eto.Forms.Control.ShownEvent:
-					Control.Shown += Connector.HandleShown;
+					Control.Shown += Connector.HandleShownEvent;
+					break;
+				case Window.OwnerChangedEvent:
```

Comment:
If it isn't handled in the Platform-Handling, there's an exception of a missing event. That was the reason I didn't used the Properties-Event handling in f5d7b493c14c8f7b60ce2e358ec9982c8fa64dd3